tst/LargeAlgebras.tst

```diff
@@ -0,0 +1,83 @@
+gap> START_TEST( "LargeAlgebras.tst");
+
+#
+gap> QG:=GroupRing(Rationals,SmallGroup(1920,236000));;
+gap> W:=WedderburnDecompositionInfo(QG);;
+gap> A:=W[Length(W)];
+[ 1, NF(5,[ 1, 4 ]), 120, [ [ 2, 11, 0 ], [ 2, 19, 60 ], [ 2, 29, 0 ], [ 2, 31, 0 ] ], [ [ 0, 0, 0 ], [ 60, 0 ], [ 0 ] ] ]
+gap> LocalIndicesOfCyclotomicAlgebra(A);
+[ [ infinity, 2 ] ]
+
+#
+gap> QG:=GroupRing(Rationals,SmallGroup(1920,236001));;
+gap> W:=WedderburnDecompositionInfo(QG);;
+gap> A:=W[Length(W)];
+[ 1, NF(5,[ 1, 4 ]), 120, [ [ 2, 11, 0 ], [ 2, 19, 0 ], [ 2, 29, 60 ], [ 2, 31, 0 ] ], [ [ 0, 0, 0 ], [ 60, 0 ], [ 0 ] ] ]
+gap> LocalIndicesOfCyclotomicAlgebra(A);
+[  ]
+
+#
+gap> QG:=GroupRing(Rationals,SmallGroup(1920,236002));;
+gap> W:=WedderburnDecompositionInfo(QG);;
+gap> A:=W[Length(W)];
+[ 1, NF(5,[ 1, 4 ]), 120, [ [ 2, 11, 0 ], [ 2, 19, 60 ], [ 2, 29, 0 ], [ 2, 31, 0 ] ], [ [ 60, 0, 60 ], [ 0, 60 ], [ 0 ] ] 
+ ]
+gap> LocalIndicesOfCyclotomicAlgebra(A);
+[  ]
+
+#
+gap> QG:=GroupRing(Rationals,SmallGroup(1920,236003));;
+gap> W:=WedderburnDecompositionInfo(QG);;
+gap> A:=W[Length(W)];
+[ 1, NF(5,[ 1, 4 ]), 120, [ [ 2, 11, 0 ], [ 2, 19, 0 ], [ 2, 29, 60 ], [ 2, 31, 0 ] ], [ [ 0, 60, 0 ], [ 0, 0 ], [ 0 ] ] ]
+gap> LocalIndicesOfCyclotomicAlgebra(A);               
+[ [ 5, 2 ] ]
+
+#
+gap> QG:=GroupRing(Rationals,SmallGroup(1920,236004));;
+gap> W:=WedderburnDecompositionInfo(QG);;
+gap> A:=W[Length(W)];
+[ 8, NF(120,[ 1, 79, 91, 109 ]) ]
+
+#
+gap> QG:=GroupRing(Rationals,SmallGroup(1920,236005));;
+gap> W:=WedderburnDecompositionInfo(QG);;
+gap> A:=W[Length(W)];
+[ 8, NF(120,[ 1, 29, 91, 119 ]) ]
+gap> LocalIndicesOfCyclotomicAlgebra(A);
+[  ]
+
+#
+QG:=GroupRing(Rationals,SmallGroup(1920,236006));;
+gap> W:=WedderburnDecompositionInfo(QG);;
+gap> A:=W[Length(W)];
+[ 8, NF(120,[ 1, 71, 91, 101 ]) ]
+gap> LocalIndicesOfCyclotomicAlgebra(A);
+[  ]
+
+#
+gap> QG:=GroupRing(Rationals,SmallGroup(1920,236007));;
+gap> W:=WedderburnDecompositionInfo(QG);;
+gap> A:=W[Length(W)];
+[ 1, NF(5,[ 1, 4 ]), 120, [ [ 2, 11, 0 ], [ 2, 19, 0 ], [ 2, 29, 60 ], [ 2, 31, 0 ] ], [ [ 0, 0, 0 ], [ 60, 0 ], [ 60 ] ] ]
+gap> LocalIndicesOfCyclotomicAlgebra(A);
+[ [ 5, 2 ] ]
+
+#
+gap> QG:=GroupRing(Rationals,SmallGroup(1920,236008));;
+gap> W:=WedderburnDecompositionInfo(QG);;
+gap> A:=W[Length(W)];
+[ 1, NF(5,[ 1, 4 ]), 120, [ [ 2, 11, 0 ], [ 2, 19, 0 ], [ 2, 29, 0 ], [ 2, 31, 0 ] ], [ [ 0, 0, 0 ], [ 0, 60 ], [ 0 ] ] ]
+gap> LocalIndicesOfCyclotomicAlgebra(A);
+[ [ infinity, 2 ] ]
+
+#
+gap> QG:=GroupRing(Rationals,SmallGroup(1920,236009));;
+gap> W:=WedderburnDecompositionInfo(QG);;
+gap> A:=W[Length(W)];                    
```

Comment:
The whole tests is very useful, but takes a bit too much - it would be useful to think how to optimise this, for example:
- Use `A` as input, at least for some examples?
- See if some examples do not increase code coverage and can be dropped?
- Move it to additional extended test, which will be not run in Travis, but will be easily accessible to run locally (I did this for some other packages - can handle this).